### PR TITLE
17 install php 80 dependencies composer

### PR DESCRIPTION
**Pull request by JulieTENOU**

### CODE REVIEW

File: roles/install-php/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    update_cache: yes
+
+- name: Checking whether Composer is installed
+  stat:
+    path: "{{composer_install_dir}}/composer"
+  register: composer_path
+
+- name: Install Composer
+  ansible.builtin.apt:
+    name: composer
```
Comment (MelvinCou) [inline roles/install-php/tasks/main.yml:20]: Apt install Composer 2.0, which seems to have issues with php8.
Correct way to install latest version :

```yaml
- name: Get composer installer
  shell: |
    chdir=~
    curl -sS https://getcomposer.org/installer -o composer-setup.php
    HASH=`curl -sS https://composer.github.io/installer.sig`
    php -r "if (hash_file('SHA384', 'composer-setup.php') === '$HASH') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;"
- name: Install composer globally
  become: true
  shell: php composer-setup.php --install-dir=/usr/local/bin --filename=composer
```

File: roles/install-php/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+- name: install certificates
+  apt:
+    pkg: "{{php_certificates}}"
+
+- name: Add Sury PPA repo for php8.0
```
Comment (MelvinCou) [inline roles/install-php/tasks/main.yml:5]: We are using php8.2, not 8.0

File: roles/install-php/vars/main.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+php_modules: 
+      - php8.2-cli
+      - php8.2-common
+      - php8.2-mysql
+      - php8.2-zip
+      - php8.2-gd
+      - php8.2-mbstring
+      - php8.2-curl
+      - php8.2-xml
+      - php8.1-bcmath
```
Comment (MelvinCou) [inline roles/install-php/vars/main.yml:12]: Why php8.1-bcmath ? php8.2-bcmath does not exist ?

File: roles/install-php/vars/main.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+php_executable: php
+composer_version: 1.6.4
+composer_install_dir: /usr/local/bin
+php_certificates :
```
Comment (MelvinCou) [inline roles/install-php/vars/main.yml:16]: Installing composer will also need :
```yaml
- git
- unzip
```

File: roles/install-php/vars/main.yml (hunk 3 above)
Comment (JulieTENOU) [reply roles/install-php/vars/main.yml:12]: my bad, typo mistake

**Pull request merged**